src/gui/editors/AutomationEditor.cpp

```diff
@@ -1256,7 +1320,22 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 			}
 
 			break;
-		case ERASE: cursor = s_toolErase; break;
+		case ERASE:
```

Comment:
I notice this switch doesn't have `{}` around the case statements like the above. Let's add them just to be consistent.

Reply:
Added the explicit blocks for consistency and also a `break;` statement on the last case of the switch, even though it wasn't necessary. That's for preventing mistakes in the future when adding new cases.